notebooks/features/responsible_ai/Interpretability - PDP and ICE explainer.ipynb

```diff
@@ -921,6 +921,166 @@
                 "ICE plots show model behavior on individual observations. Each line represents the prediction from the model if we vary the feature of interest."
             ]
         },
+        {
+            "cell_type": "markdown",
+            "metadata": {
+                "application/vnd.databricks.v1+cell": {
+                    "inputWidgets": {},
+                    "nuid": "727db1d5-e732-41c0-8824-5b92fa6b780f",
+                    "showTitle": false,
+                    "title": ""
+                }
+            },
+            "source": [
+                "## PDP-based Feature Importance\n",
+                "\n",
+                "Using PDP we can calculate a simple partial dependence-based feature importance measure. The basic motivation is that a flat PDP indicates that the feature is not important, and the more the PDP varies, the more important the feature is. Formulas are different for numeric and categorical features. \n",
+                "\n",
+                "\n",
+                "\n",
+                "If you want to learn more please visit [this link](https://christophm.github.io/interpretable-ml-book/pdp.html#pdp-based-feature-importance)."
```

Comment:
nit: here and elsewhere please describle what the link is instead of saying "this link". Instead say something like, to learn more, check out [Christoph M's Interpretable ML Book](...) makes it a bit more predictable ;)